### ISSUE TITLE
Ability to edit visualizer (enhancement)

### ISSUE THREAD

**Issue by cu1tur4l** [abh80/smart-edge#60]
First, has the visualizer been changed to oscilloscope? If not, then issue can be closed, I think.

If yes, then is there a possibility to have an option to change it to how it was? Or even better, visually leave it as it is, but change the oscilloscope to visualizer. 

I don't know if the terminology I use is correct, so I will explain what I mean using this video:
https://user-images.githubusercontent.com/102048095/196823380-39536bc1-b145-46f1-be08-39791a2d7f4d.mp4
I think earlier the app worked like the bottom visualizer, and now it works more like the oscilloscope one on the right.

**Comment by chase-haskell** [comment]
Yeah, I think it would be nice to change the number and size of visualizer bars
